modules/ding_adgangsplatformen/ding_adgangsplatformen.module

```diff
@@ -434,23 +434,38 @@ function ding_adgangsplatformen_callback() {
         if (_ding_adgangsplatformen_provider_login($user_info)) {
           // Once the user is logged in then try to migrate all variations of
           // the user which might exist on the current site.
-          if (module_exists('ding_react') && is_array($user_info['attributes']['agencies'])) {
-            $user_agencies = $user_info['attributes']['agencies'];
+          if (module_exists('ding_react')) {
```

Comment:
I'm not sure that I understand why code related to list migration is placed inside ding_adgangsplatformen. Why not introduce a `hook_ding_adgangsplatformen_post_login()` so this module can been kept clean?

Reply:
I have added a new hook `ding_adgangsplatformen_login` and implemented this in `ding_react`. I opted for this name over `ding_adgangsplatformen_post_login` because it closer resembles the existing `hook_user_login`.

There is currently also a function called `ding_adgangsplatformen_login()` but that is not called when invoking the hook.